test/ci/launch.yml

```diff
@@ -0,0 +1,31 @@
+---
+- hosts: localhost
+  connection: local
+  gather_facts: false
+  tasks:
+    - include_vars: "vars.yml"
+
+    - include_tasks: create_aws_vm.yml
+      register: aws
+      with_items: "{{ vms }}"
+      loop_control:
```

Comment:
There's no reason to loop over this file.  ec2 module can make an arbitrary number of vms, we can just loop on that task.

Reply:
I wanted to give them unique names so that the host could be found easily in the EC2 console

Reply:
This won't work that easily.

1. `aws_ami_tags` would have to be global, currently its possible to set different tags for each node group. This is probably fine, as `aws_image` per group would work
2. if we use `count: n` here there'd be n duplicate `-master` machines, which would create duplicate names in the inventory

I think its fine to leave it like this until a switch to a proper dynamic inventory